mqsf/job_factory.py

```diff
@@ -26,14 +26,14 @@ def create_job(self, job_config):
 
         if not plugin_name:
             raise MQSFJobException(
-                'No plugin type provided, cannot create job.'
+                'No plugin type provided, cannot create job'
             )
 
         job_plugin = self.plugin_manager.get_plugin(name=plugin_name)
 
         if not job_plugin and not self.can_skip:
```

Comment:
may be this could be combined with the following if and create 2 nested ifs?

Reply:
I bounced between both implementations a couple times and ended up going with a flatter structure. I think the flatter structure is worth it from readability standpoint vs the tiny overhead of doing the "not job_plugin" check twice.